blog/_posts/2022-01-25-helm-improvements.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: "helm-improvements"
+date: 2022-01-25
+author: nick
+layout: blog
+title: "The More YAML You Have, The More You Should Consider Helm"
+subtitle: "An announcement of improvements to Tilt <> Helm interop, and a discussion on the right time to use Helm."
+description: "An announcement of improvements to Tilt <> Helm interop, and a discussion on the right time to use Helm."
+image: "/assets/images/helm-improvements/elephant-seals.jpg"
+image_caption: "The <a href='https://elephantseal.org/'>elephant seals of San Simeon</a> are a good demonstration of what it looks like when your YAML gets out of control."
+tags:
+  - helm
+  - yaml
+---
+
+There are two major ways we see teams use Helm charts in dev environments.
+
+- You want to use an off-the-shelf server. You found an existing chart for it on [Artifact Hub](https://artifacthub.io/)!
+
+- You're iterating on a server that has so much YAML that you need Helm to get your arms around it.
+
+We've made some big improvements to our Helm support recently with the [`helm_resource`
+extension](https://github.com/tilt-dev/tilt-extensions/tree/master/helm_resource).
+
+Let's dig into how you'd use it and why.
+
+## Why Helm?
+
+Helm is a packaging standard for Kubernetes apps.
+
+In its simplest form, a Helm chart is simply a collection of Kubernetes objects
+described in YAML, with templating support to adapt those objects to different
+environments. Your objects might look slightly different in prod vs in dev.
+
```

Comment:
This might be outside the scope of this blog post, but helm can also manage the entire lifecycle of a release, i.e. deleting resources when calling `helm uninstall` which is a big plus point over deploying resources only via `kubectl`

Reply:
added a note on this! ya, i went back and forth into how much i should bother explaining helm in this blog post. But i think it's good to have a paragraph explaining the thrust of older features vs newer features.